cryptoapi_windows.go

```diff
@@ -138,7 +138,7 @@ func injectCertCryptoApi(derBytes []byte) {
 	// Windows CryptoAPI uses the SHA-1 fingerprint to identify a cert.
 	// This is probably a Bad Thing (TM) since SHA-1 is weak.
 	// However, that's Microsoft's problem to fix, not ours.
-	fingerprint := sha1.Sum(derBytes)
+	fingerprint := sha1.Sum(derBytes) //nolint:gosec
```

Comment:
As above, please suppress a specific `gosec` rule rather than the entire `gosec` linter.